engine/src/main/java/org/terasology/world/block/family/FreeFormSupported.java

```diff
@@ -20,8 +20,12 @@
 import java.lang.annotation.RetentionPolicy;
 import java.lang.annotation.Target;
 
+/**
+ * Annotation used to mark if a {@link BlockFamily} supports blocks that have freeform shapes.
```

Comment:
Where is described what a _ freeform_ is? Should there be a link to that (`@see` could be used for such cross references)

Reply:
I'm not quite sure what freeform is.